### ISSUE TITLE
New Jukebox songs

### ISSUE THREAD

**Issue by Crafter215** [studio-minus/ppg-bugs-and-requests#2415]
**Feature description**
Add new Jukebox songs to the base game, such as:

Waltz of the Flowers: https://www.youtube.com/watch?v=DiRRfv1Oyt4
Nocturne, Op 9 No 2: https://www.youtube.com/watch?v=yOGfiMpqZMk

**Feature purpose**
Would add a lot to the atmosphere of the game if there were more jukebox songs and it would surely. Furthermore, I do think that they fit in with the other Jukebox songs currently in the game perfectly.

**Additional context**
A feature request was made a while ago on the same topic, but no copyright free music was linked and thus the request was forgotten about.

**Comment by ATPGaming** [comment]
Duplicate or related to
#2144

**Comment by DindinYT37** [comment]
> "A feature request was made a while ago on the same topic, but no copyright free music was linked and thus the request was forgotten about."
> 
> You didn't need to put it like that, holy shit. I was linking public domain music for your information, just not the recordings themselves, the compositions are safe to use.

He's right, though.

**Comment by TeslaCoil9001** [comment]
> He's right, though. I mean – he speaks like a Skyrim NPC, but he's right.

Now every time ATPGaming talks I'm gonna imagine Skyrim music

**Comment by Crafter215** [comment]
@ATPGaming 

> You didn't need to put it like that, holy shit. I was linking public domain music for your information, just not the recordings themselves, the compositions are safe to use.

Right, and I linked 2 different pieces where the pieces don't have copyright from the creator. So what's your point? Didn't mean to personally offend you lol. I know that the compositions are safe to use.

**Comment by Crafter215** [comment]
It's more likely to get added if there's less effort required to add it. If the music from the link is already copright free zooi is able to download them using a youtube downloader and just use the mp4 file directly.

**Comment by ATPGaming** [comment]
> @ATPGaming
> 
> > You didn't need to put it like that, holy shit. I was linking public domain music for your information, just not the recordings themselves, the compositions are safe to use.
> 
> Right, and I linked 2 different pieces where the pieces don't have copyright from the creator. So what's your point? Didn't mean to personally offend you lol. I know that the compositions are safe to use.

I will admit that I overreacted. I just found a link on Google and pasted it in because I was just showing what the composition sounded like, I didn't know that it would matter if the recording itself was safe to use

> It's more likely to get added if there's less effort required to add it. If the music from the link is already copright free zooi is able to download it using a youtube downloader

You are correct
